x-pack/plugins/observability_solution/apm/server/routes/errors/get_error_groups/get_error_group_main_statistics.ts

```diff
@@ -129,16 +139,14 @@ export async function getErrorGroupMainStatistics({
             sample: {
               top_hits: {
                 size: 1,
-                _source: [
+                fields: asMutableArray([
                   TRACE_ID,
-                  ERROR_LOG_MESSAGE,
-                  ERROR_EXC_MESSAGE,
-                  ERROR_EXC_HANDLED,
-                  ERROR_EXC_TYPE,
                   ERROR_CULPRIT,
+                  AT_TIMESTAMP,
                   ERROR_GROUP_ID,
-                  '@timestamp',
-                ],
+                  ERROR_ID,
+                ] as const),
+                _source: [ERROR_LOG_MESSAGE, ERROR_EXC_MESSAGE, ERROR_EXC_HANDLED, ERROR_EXC_TYPE],
```

Comment:
@dgieselaar why do those still need to be from `_source`? 
`ERROR_LOG_MESSAGE, ERROR_EXC_MESSAGE, ERROR_EXC_HANDLED, ERROR_EXC_TYPE` are all available as single-value fields